README.md

```diff
@@ -59,6 +59,23 @@ make build
 This command executes the Scarb build process, resulting in the creation of a
 Sierra program.
 
+## Test Project
+
+Run the following command to run all tests:
+
+```bash
+make test
+```
+
+Or you can filter by test keyword,
+
+For example the following will display only tests related to the "Mint" functionality:
+
+```bash
+make test FILTER=Mint
```

Comment:
Is not clear if this filters for the name of the test or mod of tests. Could you provide more information and a more detailed example?